apps/src/lab2/views/components/Instructions/NavigationArea.tsx

```diff
@@ -37,6 +39,8 @@ interface NavigationAreaProps {
   styleAsBubble?: boolean;
   /** Optional on continue/finish callback. */
   onContinue?: () => void;
+  /** Whether or not to keep the text 'simple' */
```

Comment:
Nit: could rename to something like `buttonTextVariant`.

Reply:
That's fair! I do still think a more specific property name would be helpful though as `NavigationArea` refers to more than just the navigation button.